src/github.com/getlantern/flashlight/status_pages/generic_error.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Lantern: Address not found</title>
```

Comment:
Same here in terms of making it generic -- instead of "Address not found" maybe "Error Accessing Page"?